setup.py

```diff
@@ -36,7 +36,7 @@
     'scripts': glob.glob('scripts/*'),
     'version': version.version,
     'include_package_data': True,
-    'install_requires': reqs,
+    'install_requires': ['numpy>=1.15', 'scipy', 'astropy>=4.0', 'pyyaml', 'pyuvdata'],
```

Comment:
If we're gonna have the `requirements.txt` file (which I am agnostic to), then it's probably better just to load that file in here (as it used to be). I know it's less explicit, but it's pretty easy to see where it's being read from, and the file is easily human-readable. Otherwise, we could just remove the requirements.txt as well.

Reply:
I will just remove the requirements.txt file, I find that approach less readable.

Reply:
I am also in favor of removing the requirements.txt, but we should keep two things in mind:
-The requirements.txt file was a request from the JOSS reviewer a year ago. I guess he preferred installing with pip?
-There is a line in the README.md (under installation) that references it.

Reply:
What the JOSS reviewer wanted was for the installation to be automatic with pip. We have achieved that (except for pyradiosky) with another approach so I don't think the requirements file is really needed. We should push on a pip package for pyradiosky so that we can make it easier to install pyuvsim. Thanks for pointing out the required readme change.

Reply:
Ok, I removed it and updated the readme and setup.py. I also added instructions for installing pyradiosky, which can't be done with just the `pip install` step. Please take a look and let me know if I've missed anything.

Reply:
Just realized I forgot about installing the pre-commit hook, just added that as well.